### PR TITLE
ref(crons): Clarify next_expected vs next_expected_latest

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/monitors/models.py [hunk 1]
```diff
@@ -522,7 +527,8 @@ def mark_failed(
         elif reason == MonitorFailure.DURATION:
             new_status = MonitorStatus.TIMEOUT
 
-        next_checkin = self.monitor.get_next_scheduled_checkin(next_checkin_base)
+        next_checkin = self.monitor.get_next_expected_checkin(next_checkin_base)
+        next_checkin_latest = self.monitor.get_next_expected_checkin_latest(next_checkin_base)
```
Comment (rjo100) [inline src/sentry/monitors/models.py:531]: the reason i added that helper function was to not have to call `get_next_expected_checkin` twice, but not sure if there's a cleaner way to do it

Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:531]: ohhhh, doy. Makes sense

Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:531]: Are you worried about performance here?

Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:531]: I think I'm not too worried about the performance of this, let's keep it like this actually, it's a little more obvious what's going on I think

Comment (rjo100) [reply src/sentry/monitors/models.py:531]: i'm not worried either but just felt off to do it twice in a row

**Review by rjo100 — APPROVED** [review]

**Pull request merged**